data/iamaamunir.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Munir Abdullahi",
+  "bio": "Technical Writer | Backend Engineer | Emerging DevOps Enginer",
+  "links": [
+    {
+      "group": "My Links",
+      "name": "Hashnode"
```

```suggestion
      "name": "Hashnode",
```
there is a comma missing here